R/coerce_to_messydate.R

```diff
@@ -16,27 +17,31 @@
 as_messydate <- function(x) UseMethod("as_messydate")
 
 #' @describeIn as_messydate Coerce from `Date` to `messydt` class
+#' @return A messydt class object
```

Comment:
I think these additional `@return`s are redundant (and might lead to duplication in the help/website documentation file